common.mk

```diff
@@ -36,13 +36,20 @@ TESTCHIPIP_CLASSES ?= "$(TESTCHIP_DIR)/target/scala-$(SCALA_VERSION_MAJOR)/class
 # jar creation variables and rules
 #########################################################################################
 FIRRTL_JAR := $(base_dir)/lib/firrtl.jar
+FIRRTL_TEST_JAR := $(base_dir)/test_lib/firrtl-test.jar
 
 $(FIRRTL_JAR): $(call lookup_srcs,$(CHIPYARD_FIRRTL_DIR),scala)
 	$(MAKE) -C $(CHIPYARD_FIRRTL_DIR) SBT="$(SBT)" root_dir=$(CHIPYARD_FIRRTL_DIR) build-scala
 	mkdir -p $(@D)
 	cp -p $(CHIPYARD_FIRRTL_DIR)/utils/bin/firrtl.jar $@
 	touch $@
 
+$(FIRRTL_TEST_JAR): $(call lookup_srcs,$(CHIPYARD_FIRRTL_DIR),scala)
+	mkdir -p $(@D)
```

Comment:
I think we should match the other jar creation order of commands.
I don't really care whether its one way or the other. It seems like in the order you've written sometimes test_lib is created and empty (not really a big deal).

Reply:
Will do.